README.md

```diff
@@ -1,3 +1,58 @@
-# Rules Distroless
+# Bazel rules for fetching Debian packages
 
-Core tooling for building distroless images
+This ruleset designed to replace commands such as `apt-get install`, `passwd`, `groupadd`, `useradd`, `update-ca-certificates`.
+
+> [!NOTE]
+> rules_distroless is an beta software and doesn't have a stable Public API yet, however many are already using it in production.
+>
+> See [Adopters](#adopters) section to see who's already using it.
+
+# Usage
+
+Our [examples](/examples) demonstrate how to accomplish typical tasks such as <b>create a new user group</b> or <b>create a new home directory</b>.
+
+- [groupadd](/examples/group)
+- [passwd](/examples/passwd)
+- [useradd --home](/examples/home)
+- [update-ca-certificates](/examples/cacerts)
+- [keytool](/examples/java_keystore)
+- [apt-get install](/examples/debian_snapshot) <i>from Debian repositories.</i>
+- [apt-get install](/examples/ubuntu_snapshot) <i>from Ubuntu repositories.</i>
+
+We also we have distroless-specific rules that could be useful 
+
+- [flatten](/examples/flatten): <i>flatten multiple `tar` archives.</s>
+- [os_release](/examples/os_release): <i>create a `/etc/os-release` file</s>
+- [locale](/examples/locale): <i>strip `/usr/lib/locale` to be smaller.</s>
+- [dpkg_statusd](/examples/statusd): <i>creates a package database at /var/lib/dpkg/status for scanners to discover installed packages.</i>
```

Comment:
```suggestion
- [dpkg_statusd](/examples/statusd): <i>creates a package database at /var/lib/dpkg/status.d for scanners to discover installed packages.</i>
```

Reply:
or is this something that is distroless specific only (modified for distroless?)

Reply:
We support both formats though, distroless specific one and the apt generates. so this is fine